system/alternatives.py

```diff
@@ -69,6 +76,7 @@ def main():
             name = dict(required=True),
             path = dict(required=True),
             link = dict(required=False),
+            priority = dict(required=False),
```

Comment:
I think this may be a bit cleaner if you remove the `DEFAULT_LINK_PRIORITY` code and just make this:

``` python
priority = dict(required=False, type='int', default=50)
```

Then you can remove the conditional below (L88-91) as well.